src/views/GovernanceReviewTeam/GRBReview/PresentationLinksCard.tsx

```diff
@@ -0,0 +1,254 @@
+import { SystemIntakeDocumentStatus } from 'types/graphql-global-types';
+
+import ITGovAdminContext from '../ITGovAdminContext';
+
+export type PresentationLinksCardProps = {
+  systemIntakeID: string;
+  grbPresentationLinks: SystemIntake['grbPresentationLinks'];
+};
+
+function PresentationLinksCard({
+  systemIntakeID,
+  grbPresentationLinks
+}: PresentationLinksCardProps) {
+  const { t } = useTranslation('grbReview');
+
+  const isITGovAdmin = useContext(ITGovAdminContext);
+
+  const { showMessage } = useMessage();
+
+  const {
+    recordingLink,
+    recordingPasscode,
+    transcriptFileStatus,
+    transcriptFileURL,
+    presentationDeckFileStatus,
+    presentationDeckFileURL
+  } = grbPresentationLinks || {};
+
+  const isEmpty =
+    grbPresentationLinks === null ||
+    (grbPresentationLinks &&
+      recordingLink === null &&
+      transcriptFileStatus === SystemIntakeDocumentStatus.UNAVAILABLE &&
+      presentationDeckFileStatus === SystemIntakeDocumentStatus.UNAVAILABLE);
+
+  // Remove links handling
+
+  const [deleteSystemIntakeGRBPresentationLinks] =
+    useDeleteSystemIntakeGRBPresentationLinksMutation({
+      variables: {
+        input: {
+          systemIntakeID
+        }
+      }
+    });
+
+  const [
+    isRemovePresentationLinksModalOpen,
+    setRemovePresentationLinksModalOpen
+  ] = useState<boolean>(false);
+
+  const removePresentationLinks = () => {
+    deleteSystemIntakeGRBPresentationLinks()
+      .then(() => {
+        showMessage(t('asyncPresentation.modalRemoveLinks.success'), {
+          type: 'success'
+        });
+      })
+      .catch(() => {
+        showMessage(t('asyncPresentation.modalRemoveLinks.error'), {
+          type: 'error'
+        });
+      })
+      .finally(() => {
+        setRemovePresentationLinksModalOpen(false);
+      });
+  };
+
+  // External link modal handling
+
+  const [externalUrl, setExternalUrl] = useState<string>('');
+  const [isExternalModalOpen, setExternalModalOpen] = useState<boolean>(false);
+
+  const externalModalScopeRef = useRef<HTMLDivElement>(null);
+
+  function linkHandler(event: MouseEvent) {
+    const a = (event.target as HTMLElement)?.closest(
+      '.presentation-card-links a'
+    );
+    if (a) {
+      event.preventDefault();
+      const href = a.getAttribute('href');
+      if (href) {
+        setExternalUrl(href);
+        setExternalModalOpen(true);
+      }
+    }
+  }
+
+  useEffect(() => {
+    const eventEl = externalModalScopeRef.current;
+    eventEl?.addEventListener('click', linkHandler);
+    return () => {
+      eventEl?.removeEventListener('click', linkHandler);
+    };
+  }, []);
+
+  // Render empty if not an admin and no links
+  if (!isITGovAdmin && isEmpty) return null;
+
+  return (
+    <>
+      {/* Asynchronous presentation links card */}
+      <div className="usa-card__container margin-left-0 border-width-1px shadow-2 margin-top-3 margin-bottom-4">
+        <CardHeader>
```

Comment:
The margin between `CardHeader` and `CardBody` needs to be decreased to match the Figma.